recipe/meta.yaml

```diff
@@ -42,49 +46,50 @@ source:
       - patches/0031-work-around-for-warning-that-clang-falsely-treats-as.patch
       - patches/0001-Hardcode-BUILD_PREFIX-in-build_pip_package.patch
       - patches/0001-Only-link-to-sparse_core_layout_proto_cc-headers.patch
+      - patches/0032-loosen-hermetic-python.patch
   - url: https://github.com/tensorflow/estimator/archive/refs/tags/v{{ estimator_version.replace(".rc", "-rc") }}.tar.gz
     sha256: 2d7e100b1878084da34b5e23b49a0cbb5ee8a7add74b7dd189a82ada1cf85530
     folder: tensorflow-estimator
 
 build:
-  number: 0
-  skip: true  # [win]
+  number: {{ build }}
+  skip: true  # [win or s390x]
   skip: true  # [python_impl == 'pypy']
   skip: true  # [py<39]
```

Comment:
might want to make a note that it's untested with python 3.13 in case someone tries to rebuild this for that https://www.tensorflow.org/install/source#tested_build_configurations

Reply:
As I understand, their first 3.13 version will be tensorflow 2.19. I don't think this would build with 3.13.

Reply:
ok, thanks, then a skip py>312 perhaps?